src/api/app/models/history_element/request_review_added.rb

```diff
@@ -1,12 +1,16 @@
 module HistoryElement
   class RequestReviewAdded < HistoryElement::Request
-    # self.description_extension is review id
     def description
       'Request got a new review request'
     end
 
     def user_action
-      'added review'
+      "added <strong>#{review.reviewed_by}</strong> as a reviewer".html_safe
```

Comment:
It's totally out of scope for this PR, but this will have to be refactored one day. This code doesn't belong in a model, it's view code. Could you please create a card in the request workflow redesign backlog for this? It should be a refactoring of all HistoryElement models and the views where they are used.

Reply:
@dmarcoux : for the time being, until they get refactored them all, what do you think about the following implementation to get rid of the `html_safe` call and still have the additional view output? https://github.com/openSUSE/open-build-service/pull/13227/commits/9d388407cb0e11fe5e9d3d87892f69350e8908b9 (CC-ing @hennevogel)

Reply:
It's good enough for now. This will be refactored in the future anyway.